libs/frontend/feature-render-builder/src/lib/render-builder.ts

```diff
@@ -225,32 +230,54 @@ export const computePropsContent = (
 
   const { editor } = site
   const isSelected = editor?.selectedComponent?.id === component.id
+  const reusableCmp = resolveReusableComponent(
```

Comment:
Yeah, I think it makes sense!
I think there's also some data (mainly ids) that has to be modified when rendering a reusable instance. For example, the component id in `cmp.style.overrides` in a reusable component should be changed to the instance's children ids. We might have to do the same thing to the children ids in custom behaviors too, if there's any.

Reply:
I'm not really sure how to handle child style overrides correctly. We want to be able to make style edits to the original/reusable component and have them show up in the instances. Since the IDs will be different, that will be pretty complicated and inefficient.

Maybe the best solution is to disable child override styles in reusable components? The main use case I've found so far for style overrides is to affect the hover style of children. It's occasionally useful but might not be a big problem to disable, and could always be added later if we come up with a clever implementation. Another option could be to only duplicate the overrides when a new instance is created, so the compromise is that changes to the reusable component style overrides aren't propagated.